Islandora-CLAW/islandora_demo
# adding optional field label translations

**GitHub Issue**: (link)
* https://github.com/Islandora-CLAW/CLAW/issues/939
* https://github.com/Islandora-CLAW/CLAW/pull/1093

# What does this Pull Request do?
This PR adds a tamil.po file.  When imported, it will translate the current Repository Item metadata field lables and field hints into Tamil. 

# How should this be tested?
* https://github.com/Islandora-CLAW/CLAW/pull/1093 describes steps to set up a multilingual site.  Choose Tamil as the second language.  
* Enable "Configuration translation"
* Go to `http://localhost:8000/admin/config/regional/translate/import`
* Import the tamil.po file in this PR, check all the config options for import (i.e Treat imported strings as custom translations)
* Clear the cache
* Create a Repositiry Item
* Click to translate
* You should see all the field lables for Repository Items be translated into another language (in this case Tamil)

# Additional Notes:


# Interested parties
@kanasznagyzoltan
@Islandora-CLAW/committers
